internal/cli/default_setter_opts.go

```diff
@@ -274,21 +279,5 @@ func (opts *DefaultSetterOpts) DefaultQuestions() []*survey.Question {
 			},
 		},
 	}
-	if opts.IsCloud() {
-		defaultPath := config.MongoShellPath()
-		if defaultPath == "" {
-			defaultPath = mongosh.Path()
-		}
-		atlasQuestion := &survey.Question{
-			Name: "mongoShellPath",
-			Prompt: &survey.Input{
-				Message: "Default MongoDB Shell Path:",
-				Help:    "MongoDB CLI will use the MongoDB shell version provided to allow you to access your deployments.",
-				Default: defaultPath,
-			},
-			Validate: validate.OptionalPath,
-		}
-		q = append(q, atlasQuestion)
-	}
```

Comment:
Does this mean we now never prompt for the path? What if it wasn't detected in the standard path, but it is installed for instance?

I'm wondering does this put us in a situation where if we can't detect the path, the user can't enter it.

Reply:
* `Does this mean we now never prompt for the path?` Yes
* `What if it wasn't detected in the standard path, but it is installed for instance?` It would be updated during login/atlas config commands as we call `SetUpMongoSHPath()` as part of these commands
* `I'm wondering does this put us in a situation where if we can't detect the path, the user can't enter it.` that is a great point, we would be relying on finding the path or the customer can edit the configuration file directly. One thing I can suggest would be to ask the path if it's not detected for one of the commands (maybe config init?) but still it would get automatically updated if we find a new path...

Reply:
One more thing I missed to explain is that for quickstart we ask 1) `"Do you want to provide the path to your MongoDB Shell binary?"` and if so, then we offer them the option to add the mongo path, I did not change this part as it is hidden under another question.

Reply:
>"Do you want to provide the path to your MongoDB Shell binary?"

Ah ok, at least they are getting prompted to set the path. 
[idea] How about we show the detected path if they say no?

Reply:
sure! we can do that, thank you

Reply:
actually, we only ask this if the MongoDB shell path is not present. In that sense, I think if they say no there is no way for us to say a path because we already checked config and searched for a path. wdyt?

Reply:
Yes, the path for yes allows a path to be entered.
The path for no sounds slightly more subtle. In that case, since we don't have a shell, we probably finish creating the cluster as usual but not try to call the shell to connect to the cluster, instead displaying the connection string.